pages/07.Addons/03.metakube-backups/default.en.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Velero Backups'
+published: false
+date: '22-07-2019 19:00'
+taxonomy:
+    tag:
+        - metakube
+        - backup
+        - velero
+---
+
+The Velero Backups addon installs [Velero](https://velero.io/), a fully functional Kubernetes backup solution that you can use to safely backup and restore Kubernetes cluster resources and persistent volumes.
+
```

Comment:
We should link to the tutorial and the documentation. Also we should add that the addon creates a ObjectStorage bucket and configures this as a default backup location in velero.